src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -90,23 +82,51 @@ export class ItemEditAdvancedParametersComponent implements OnInit, OnChanges {
     this.parentForm?.get('duration')?.updateValueAndValidity();
   }
 
+  onEnteringTimeMinEnabledChange(event: boolean): void {
+    if (!event) {
+      return;
+    }
+
+    const enteringTimeMin = this.parentForm?.get('entering_time_min')?.value as Date;
+
+    if (enteringTimeMin.getTime() === new Date(DEFAULT_ENTERING_TIME_MIN).getTime()) {
+      this.parentForm?.get('entering_time_min')?.patchValue(
+        new Date()
+      );
+
+      this.computeEnteringTimeMaxValue();
```

Comment:
I am not sure it is useful in your implementation:
- if max is not enabled, it will not enter into the condition on line 113, so will not do anything
- if max was already enabled, it has already been set to "now" when enabled, so it will not enter into the condition on line 113, so will not do anything

Reply:
Fixed